src/audio/clap/event_list.cpp

```diff
@@ -115,10 +112,13 @@ void EventList::AddMidiSysExEvent(const std::vector<uint8_t>& msg,
 	ev.buffer          = sysex_data.data() + sysex_data.size();
 	ev.size            = msg.size();
 
+	// We reserved `MaxMidiSysExBytes` for `sysex_data` to avoid
+	// reallocations, so make sure we never trigger a realloc here.
+	assert(sysex_data.size() + msg.size() <= MaxMidiSysExBytes);
```

Comment:
I prefer using a vector so I don't need a separate `pos` variable to keep track of.